modules/learn/pages/security/roles.adoc

```diff
@@ -1943,6 +1943,59 @@ It allows the reading of bucket settings.
 ^| image:introduction/no.png[]
 |===
 
+
+[#query-sequential-scan]
+== Query Use Sequential Scan
+
+The *Query Use Sequential Scan* role (a Query & Index role) allows users to perform a sequential scan query.
```

Comment:
It isn't "& Index". 
This reads as a type of query rather than an access method used in a query, to me.  I'd have something more like:
```
... allows users to execute a query that uses a sequential scan to access the keyspace.
```

Reply:
"Query & Index" here is referring to where tis role appears in the Web Console's  UI. You have to drop down the Query & Index category to find Query User Sequential Scans.

Reply:
Ack.

Reply:
I changed it a bit to say that it appears under "Query & Index" in the Web Console's list of roles. Hopefully, that's a bit clearer than just saying it's a such-and-such role.